peps/api/index.rst

```diff
@@ -102,3 +105,88 @@ illustrating some of the possible values for each field:
        "url": "https://peps.python.org/pep-3124/"
      }
    }
+
+release-cycle.json
+------------------
+
+There is a read-only JSON document of Python releases since version 2.6
+available at https://peps.python.org/api/release-cycle.json.
```

Comment:
I'd prefer not to commit to 'since 2.6', as at some point we should drop long-obsolete versions from the file (2.6 can now learn to drive!)